### ISSUE TITLE
Tag support for polls

### ISSUE THREAD

**Issue by kenorb** [telegramdesktop/tdesktop#5666]
### Steps to reproduce
1. Create a poll with a #tag

Here is the example: https://t.me/paranormal_dimension/2323

### Expected behaviour

The tag should be clickable, so it's easier to find.

### Actual behaviour

The tag is not clickable.

### Configuration

Windows 10

**Version of Telegram Desktop:** 1.5.9

**Used theme**: Dark

**Comment by Aokromes** [comment]
afaik, numbers cannot be tagged.

**Comment by kenorb** [comment]
> afaik, numbers cannot be hashed

You're right, it's described in another issue at #6014. I've clarified that the above. However this one is about having regular tags in the pool's description

**Comment by kenorb** [comment]
> We therefore assume that the user has lost interest or resolved the problem on their own.

No, the problem didn't resolved on their own.

**Comment by kenorb** [comment]
> Hey there!
> 
> This issue was inactive for a long time and will be automatically closed in 30 days if there isn't any further activity. We therefore assume that the user has lost interest or resolved the problem on their own.
> 
> Don't worry though; if this is an error, let us know with a comment and we'll be happy to reopen the issue.
> 
> Thanks!

Why?
